### DISCUSSION TITLE
Quizlet import support

### DISCUSSION THREAD

**Discussion by Epicheirema** [tema6120/ForgetMeNot#47] [Ideas]
Hi, I was wondering if it would be possible for someone to add support for importing card sets from quizlet. I think this app is really well designed but I currently have 100s of cards in quizlet and can't seem to import them.

**Comment by tema6120** [comment]
Ok. I explored this service and found out that on their site you can export text of cards.
<img alt="Webp net-resizeimage (15)" src="https://user-images.githubusercontent.com/34657415/111536577-177d2b80-8773-11eb-99f2-c74c27e37b96.png">
<img alt="Webp net-resizeimage (16)" src="https://user-images.githubusercontent.com/34657415/111536650-2f54af80-8773-11eb-90bd-c1c960ff2034.png">



Copy text into new file with extension .tsv (if you select Tab delimiter). Then import this file into ForgetMeNot. Make sure you select 'Tab text' format while importing:
![Webp net-resizeimage (14)](https://user-images.githubusercontent.com/34657415/111536313-c40add80-8772-11eb-9a65-aa709272f88e.png)

**Comment by Epicheirema** [comment]
Okay, perfect. Thanks a ton.

-------- Original Message --------
